packages/react/src/radioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,75 @@
+import * as RadixLabel from "@radix-ui/react-label";
+import * as RadixRadio from "@radix-ui/react-radio-group";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Flex } from "../flex";
+import { Text } from "../text";
+import * as styles from "./RadioGroup.css";
+
+type optionProps = {
```

Comment:
```suggestion
type OptionProps = {
```

type names should always be `PascalCase`

Reply:
let's add lint rule